docs/whats-new/preview-4.md

```diff
@@ -578,8 +578,80 @@ This release we've enabled a Visual Studio publish experience for .NET Aspire ap
 
 ## Azure Developer CLI
 
-The Azure Developer CLI (azd) has been updated to support the new manifest types introduced in preview 4. Azd supports prompting for parameter resources, and supports the bicep resource which enables deploying any thing can be described using a bicep module.
+The [Azure Developer CLI](https://aka.ms/azd) (azd) has been updated to support the new manifest types introduced in preview 4. Azd supports prompting for parameter resources, and supports the bicep resource which enables deploying any thing. All Azure resources can be described using a bicep module.
 
 In addition to new features, azd will automatically create secrets in Azure Container Apps for any parameter marked as a secret, and any environment variable that references a secret or uses a connection string.
 
-TODO: Expand with images
+### Prompting for parameters
+
+When azd detects a parameter.v0 resource in the manifest, it will prompt for a value. For example, consider the following code in the apphost project:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var administratorLogin = builder.AddParameter("administratorLogin");
+var administratorLoginPassword = builder.AddParameter(
+    "administratorLoginPassword", secret: true);
+
+var pg = builder.AddPostgres("postgres")
+                .AsAzurePostgresFlexibleServer(
+                    administratorLogin, administratorLoginPassword)
+                .AddDatabase("db");
+```
+
+In the preceding code, two parameters are created to represent the admin login and password. They're then passed to the `AsAzurePostgresFlexibleServer` API. This causes `azd` to ask for both parameter values.
+
+:::image type="content" source="media/preview-4/promptUser.gif" lightbox="media/preview-4/promptUser.gif" alt-text="azd parameters prompting":::
+
+To skip interactive prompting, users can set an environment variable for each parameter. For the `administratorLogin` parameter, azd looks for an environment variable with the name `AZURE_ADMINISTRATOR_LOGIN`.
```

Comment:
Does AZD magically turn the camel cased `administratorLogin` into `AZURE_ADMINISTRATOR_LOGIN`? What is the rule for this.

Reply:
Yes, azd makes it automatically.
Folks can set any name for the env var, there's no rule.